src/app/config.rs

```diff
@@ -107,6 +107,8 @@ pub struct Config {
     /// Where to talk to the control plane.
     pub destination_addr: ControlAddr,
 
+    pub trace_collector_addr: Option<ControlAddr>,
```

Comment:
To be clear... we anticipate running the collector meshed with the proxy to support identity?

Reply:
yes